### DISCUSSION TITLE
石头自清洁扫拖机器人G10S相关功能

### DISCUSSION THREAD

**Discussion by xzshow** [XiaoMi/ha_xiaomi_home#146] [Ideas]
石头自清洁扫拖机器人G10S，将房间分区后，能不能将单个房间按钮接入到HA中，比如接入到HA中是客厅按钮，按下后扫地机就打扫客厅，谢谢

**Comment by JimmyKmi** [comment]
请不要重复提问，同类问题 https://github.com/XiaoMi/ha_xiaomi_home/discussions/145
